tests/sqllogictests/src/client/clickhouse_client.rs

```diff
@@ -43,6 +43,7 @@ impl ClickhouseHttpClient {
     }
 
     pub async fn query(&mut self, sql: &str) -> Result<DBOutput> {
+        println!("Running sql with clickhouse client: [{}]", sql);
```

Comment:
Please use `debug` or `info`.

Reply:
If we use them, we'll get some noise from sqllogictest-rs.

I think those logs are basic and useful, maybe it's better to print them always.